examples/plain/scripts/upload-debug-android.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
```

Comment:
you should put [`set -e`](https://stackoverflow.com/questions/19622198/what-does-set-e-mean-in-a-bash-script) somewhere near the top of these scripts, so that if any of the commands fail, it stops and outputs the error.

When I ran this it told me the `.map` file was invalid because even though the `curl` command failed, it still wrote the output to the `.map` file. While true, this was not the source of the error so is misleading.